applications/nrf_desktop/src/modules/Kconfig.watchdog

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "Watchdog settings"
+
+config DESKTOP_WATCHDOG_ENABLE
+	bool "Enable watchdog support"
+	depends on WATCHDOG
+	help
+	  Enable watchdog to reset the device in case of problems.
+
+if DESKTOP_WATCHDOG_ENABLE
+
+config DESKTOP_WATCHDOG_TIMEOUT
+	int "Watchdog timeout value [s]"
+	default 16
+	range 16 120
+	help
+	  Timeout of watchdog timer. After the time elapse a device will be restarted.
+	  16s is the lowest possible timeout, 120s is the highest allowed value.
```

Comment:
Not sure if we really need to write about limits for timeout here (it is already defined as range)

Reply:
line removed